modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -33,12 +33,14 @@ EFAElement3D::EFAElement3D(unsigned int eid, unsigned int n_nodes, unsigned int
   if (_num_faces == 4)
   {
     _num_vertices = 4;
-    if (_num_nodes == 10)
+    if (_num_nodes == 14)
+      _num_interior_face_nodes = 4;
+    else if (_num_nodes == 10)
       _num_interior_face_nodes = 3;
```

Comment:
wait is that true

Reply:
tet10 is not 4 vertices + 6 edge vertices?

Reply:
> tet10 is not 4 vertices + 6 mid-edge nodes?

Indeed. And then you add another node for the face to form a TET14.

Reply:
so should be 0 not 3? (this is not from your PR I know)

Reply:
oh so it s on a per-face basis?

Reply:
Indeed.

Reply:
all good then